python/cuml/test/test_tsne.py

```diff
@@ -0,0 +1,100 @@
+
+
+from cuml.manifold import TSNE
+
+from sklearn.manifold.t_sne import trustworthiness
+
+import pandas as pd
+import numpy as np
+import cudf
+
+
+def test_tsne():
```

Comment:
Check how we are using `pytest.mark.parameterize` in other pytests. It's very useful to have test functions that accept arguments as input so that we also run the tests manually in the python repl as well as testing new/different inputs easily.

Reply:
Oh lol I actually deleted the parametization code. Anyways I'll put it back in